test/test_package_loading.py

```diff
@@ -105,6 +105,12 @@ def test_load_packages_sequential(selenium_standalone, packages):
     assert selenium.logs.count(f'Loading {packages[1]}') == 1
 
 
+def test_different_ABI(selenium_standalone):
+    selenium = selenium_standalone
+    selenium.load_package('numpy-dummy')
+    assert 'ABI numbers differ!' in selenium.logs
```

Comment:
Why don't we (1) copy the working numpy.js to a temporary directory (py.test has explicit support for temporary directories) and modify that file to have the wrong ABI number.  Then (2) use load_package to load it and assert that it's broken as you do here.

Reply:
Let's say the modified ``numpy.js`` [with wrong ABI] is created at ``/tmp/pytest-of-root/pytest-9/test_different_ABI_chrome_0/subdir/numpy.js'``. Do I pass this path in ``load_package``? Doing so does not work as ``load_package `` only accepts URLs ending in ``.js`` OR simply the names such as ``numpy``. How to proceed in this case?